src/common/components/Home.js

```diff
@@ -5,62 +5,102 @@ import { isEmpty } from 'lodash';
 import moment from 'moment';
 import Log from './Log';
 import Mock from './Mock';
-
+import testClass from '../testClass';
```

Comment:
I'm a bit in two minds about this util, maybe just add the test class directly to make it simpler and more readable? With the downside of having some extra useless class in the app dom but I don't see that as a big issue

Reply:
I wonder if this would be better as a babel plugin ... and when its compiled with a given config / env it simply removes it (so no function actually exists in production)!

Reply:
Another goal for next year ... write my own plugin and preset!